test_common/CMakeLists.txt

```diff
@@ -21,3 +21,8 @@ set(HARNESS_SOURCES
 )
 
 add_library(harness STATIC ${HARNESS_SOURCES})
+
+if(MSVC)
+    # Don't warn about using the portable "strdup" function.
+    target_compile_definitions(harness PRIVATE _CRT_NONSTDC_NO_DEPRECATE)
```

Comment:
I added this to the top-level cmake file a month or so ago: https://github.com/KhronosGroup/OpenCL-CTS/pull/1314

Do we need it here too?